Python/Tests/Utilities.Python.Analysis/PythonPaths.cs

```diff
@@ -255,7 +255,17 @@ public class PythonVersion {
     }
 
     public static class PythonVersionExtensions {
-        public static void AssertInstalled(this PythonVersion pyVersion, string customMessage = "Python interpreter not installed") {
+        public static void AssertInstalled(this PythonVersion pyVersion) {
+            if (pyVersion == null || !File.Exists(pyVersion.InterpreterPath)) {
+                if(pyVersion == null) {
+                    Assert.Inconclusive("pyVersion is null");
```

Comment:
the original "Python interpreter not installed" is a more useful message.

note that this is the common case. it will be very rare for pyVersion to not be null but somehow be missing its executable.

Reply:
Modified.